### ISSUE TITLE
Smooth basalt stairs, slabs. and walls have no crafting recipie

### ISSUE THREAD

**Issue by Redd56** [ConsistencyPlus/ConsistencyPlus#133]
**Versions:**
Minecraft version: 1.19.2
C+ version: 0.5.1
Fabric API / QSL / Forge version: 0.60.0

**Other mods:**
![image](https://user-images.githubusercontent.com/12189828/192163590-b42e064a-47d9-4d1e-b860-a7bebaf8d2ca.png)
all mods in the pack i made since it may be an issue with something messing up the recipies

**Logs:**
[latest.log](https://github.com/ConsistencyPlus/ConsistencyPlus/files/9641713/latest.log)

**To Reproduce:**

1.load game
2.notice no recipies

**Expected behavior:**
blocks to have recipies

**Comment by Redd56** [comment]
testing with only consistency plus right now

**Comment by Redd56** [comment]
can confirm with only consistency plus, fabric api and architechery (6.2.43)

**Comment by Cart3r1234** [comment]
Yep, turns out we only added recipes for smooth basalt gates.  I'll take care of this in a bit, thanks for reporting!
![image](https://user-images.githubusercontent.com/69820300/192170056-f2961e6b-4a9d-435b-ad1d-2d3bc84799fb.png)

**Comment by Redd56** [comment]
:skull:

**Comment by Cart3r1234** [comment]
Fixed in latest commit.  Recipes for crafting and smelting smooth basalt shapes have been properly added.

**Comment by Redd56** [comment]
Sweet cant wait for the update to drop if it hasnt, (havent checked)
